### PR TITLE
Align visually typeahead answer with other components

### LINKED ISSUE #257: Allign visually typeahead component with other components for answers

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #257

### CODE REVIEW

**Review by blcham — CHANGES_REQUESTED** [review]
See my comments.

**Comment by blcham** [comment]
also note there are some issues in chrome

File: src/components/answer/TypeaheadAnswer.jsx [hunk 1]
```diff
@@ -121,6 +121,9 @@ const TypeaheadAnswer = (props) => {
     props.onChange(option ? option.id : null);
   };
 
+  // Used if the value should not be displayed as a link
+  const valueRenderer = (children) => <>{children}</>;
```
[the remark this answers is not in the document]
Comment (LaChope) [reply src/components/answer/TypeaheadAnswer.jsx:125]: Agree with `noLinksValueRenderer`

File: src/constants/Constants.js [hunk 2]
```diff
@@ -156,6 +156,7 @@ export default class Constants {
   static BROADER = "http://www.w3.org/2004/02/skos/core#broader";
   static HAS_NON_SELECTABLE_VALUE =
     "http://onto.fel.cvut.cz/ontologies/form/has-non-selectable-value";
+  static IS_NOT_LINK = "http://onto.fel.cvut.cz/ontologies/form/is-not-link";
```
[the remark this answers is not in the document]
Comment (LaChope) [reply src/constants/Constants.js:159]: Agreed, done

File: src/components/answer/TypeaheadAnswer.jsx [hunk 3]
```diff
@@ -121,6 +121,9 @@ const TypeaheadAnswer = (props) => {
     props.onChange(option ? option.id : null);
   };
 
+  // Used if the value should not be displayed as a link
```
Comment (blcham) [inline src/components/answer/TypeaheadAnswer.jsx:124]:
```suggestion
```

**Review by blcham — APPROVED** [review]

**Pull request merged**